pkg/collectors/node_test.go

```diff
@@ -127,9 +127,10 @@ func TestNodeCollector(t *testing.T) {
 				},
 			},
 			Want: `
-        kube_node_created{node="127.0.0.1"} 1.5e+09
+		kube_node_created{node="127.0.0.1"} 1.5e+09
         kube_node_info{container_runtime_version="rkt",kernel_version="kernel",kubelet_version="kubelet",kubeproxy_version="kubeproxy",node="127.0.0.1",os_image="osimage",provider_id="provider://i-randomidentifier"} 1
-        kube_node_labels{label_type="master",node="127.0.0.1"} 1
+		kube_node_labels{label_node_role_kubernetes_io_master="",node="127.0.0.1"} 1
+		kube_node_role{node="127.0.0.1",role="master"} 1
```

Comment:
nit: Are we missing a gofmt run here?

Reply:
This is to sync with the upstream repo, if we were to make the change here we would diverge with the upstream branch.

Reply:
okay!